output/alternative/bls/CES0000000001.csv

```diff
@@ -0,0 +1,1011 @@
+EndTime,Period Start Time,Period End Time,Value
+1939-02-28 00:00:00,1939-01-01,1939-02-01,29923
```

Comment:
We usually normalize file names to lower case, believe should follow the pattern 👍
For `EndTime` could avoid writing ` 00:00:00` if it's always just a date?